pyproject.toml

```diff
@@ -1,3 +1,44 @@
+[tool.poetry]
+name = "neo4j-arrow"
+version = "0.6.0"
+description = "Neo4j Arrow Client Library"
+authors = ["Ali Ince <ali.ince@neo4j.com>"]
```

Comment:
it was already there before, but maybe set the team email in case there's any issue with the connector and you're on vacation?